### DISCUSSION TITLE
What does this mean

### DISCUSSION THREAD

**Discussion by mozile2090** [dashingsoft/pyarmor#1700] [General]
what does this does in pyarmor 

pyarmor cfg rft_auto_exclude=0

i cant understand the usefulness of this anytime i didnt run this command i get all many function been skip in the rft_trace log which means it isnt performing rft on them but when i run pyarmor cfg rft_auto_exclude=0 before gen then i got no content in the rft_trace log does this means it has  perform rft on all the skip function?

**Comment by jondy** [comment]
Check https://pyarmor.readthedocs.io/en/latest/topic/rftmode.html
